daprieu/rare-client-cool-cake
# Pp post reaction

# Description

Users can add reactions to posts.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Server side code:
```
git checkout main
git fetch --all
git checkout pp-postReaction
```

Go to a post's details page and click on a reaction. The counter should increase and the user should not be able to use the same reaction again. 

# Checklist:
